### ISSUE TITLE
If no persona, use display name

### ISSUE THREAD

**Issue by burlexpo** [bethlakshmi/GBE2#339]
We have a bunch of folks, especially volunteers, that don't have, need, or want persona.  Unfortunately, that means they don't have names.

Can we set some logic somewhere that says "If no persona, then use display name"?

**Comment by bethlakshmi** [comment]
Ugh.  Yes, but I need to know where.  This gets referenced all over the place and in some places it's feasible and in some it isn't.  If you pointed at where you mean, it would help.

**Comment by burlexpo** [comment]
Volunteer bid page -- do a search for "jvan" -- it's Josiah; but he comes up as blank (and if you view his landing page it says "Your Stuff" as opposed to "Name's Stuff).

Incidentally, it means that the volunteer alert email said 

*Hello Volunteer Reviewers!*

*has proposed to volunteer at the expo!*

*Please go to /volunteer/review to review this and other volunteer bids.*

Where normally there's a name between "Reviewers!" and "has proposed..."

**Comment by eclecticmagpie** [comment]
Why not have the persona name default to a copy of real name, with persona creation giving the opportunity to fill in a different name instead? Minimal code changes, small instructional change.

**Comment by jonkiparsky** [comment]
That's actually what happens, Michael.

@burlexpo, personas have nothing at all to do with this. There are four
users who have managed to blank out their display names in creating their
profiles. (four real users - there are also two dummy accounts in this
situation) If you want, I can set their display_names to their first+last -
this is trivial

**Comment by burlexpo** [comment]
No -- let's not worry about it now.  I need to mull this over a little.

**Comment by jonkiparsky** [comment]
Okay, mull away. fwiw, I could as easily default their display name to their username (eg, jvan) if that would be better.

**Comment by burlexpo** [comment]
Yeah, I think that would be better.  I think people are wiping out their 
'real names' because they don't want them displayed.  I'm trying to 
figure a politic way around this.

**Comment by burlexpo** [comment]
@bethlakshmi I feel like we recently created a new ticket to tackle this issue and closed it, but I can't find it.  If we did, you can close this.

**Comment by bethlakshmi** [comment]
Not entirely - #699 prevents volunteers from screwing up by keeping them from giving bad data.  It doesn't mitigate the case where someone gave us bad data.

The cases you mention;

- /volunteer/reviewlist
- /gbe - for the weirdo with no name
- the email that is sent to volunteer reviewers

will all show the behavior described on this ticket (no data) if the volunteer manages to delete the display_name.

**Comment by bethlakshmi** [comment]
We did a bunch of work on this area last year, so I think this ticket is obsolete.
